frontend/assets/style/components/_forms.scss

```diff
@@ -129,17 +130,23 @@ textarea,
     border-right: 1px solid $general_0;
   }
 
-  label {
+  .sufix {
```

Comment:
Not sure why we remove the style from label. If there is a good reason, did you had a the class to every label we use in the app?